examples/tracer-web/README.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument your JavaScript code running in the browser.
+This example uses the `ConsoleSpanExporter()` to export spans to the browser console output.
+
+## Installation
+
+```sh
+$ # from this directory
+$ npm install
+```
+
+## Run the Application
+
+```sh
+$ # from this directory
+$ npm start
+```
+
+By default, the application will run on port `8090`.
+
+To see the results, open the browser at `http://localhost:8090/` and make sure you have the browser console open. The application is using the `ConsoleSpanExporter` and will post the created spans to the browser console.
```

Comment:
Adding a link would be easier for the end users to jump on the endpoint quickly (instead of copy/paste) :)

```suggestion
To see the results, open the browser at [http://localhost:8090/](http://localhost:8090/) and make sure you have the browser console open. The application is using the `ConsoleSpanExporter` and will post the created spans to the browser console.
```

Reply:
More succinct link syntax.

```suggestion
To see the results, open the browser at <http://localhost:8090/> and make sure you have the browser console open. The application is using the `ConsoleSpanExporter` and will post the created spans to the browser console.
```